yarn.lock

```diff
@@ -677,6 +687,31 @@ locate-path@^6.0.0:
   dependencies:
     p-locate "^5.0.0"
 
+lodash._reinterpolate@^3.0.0:
```

Comment:
Hmm, looks like `sourcemap-validator` pulls in quite a few items. Not an issue, but something we might be able to upstream changes for.